build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -31,18 +31,22 @@
 
 import atexit
 import os
-import re
 import subprocess
 import tarfile
 import shutil
 import sys
+from pathlib import Path
+# Add build_tools python dir to the search path.
+sys.path.append(
```

Comment:
You most likely want `sys.path.insert(0, ...)` (you want the local path to take precedence.

Reply:
I might just be old-school, but I usually write this kind of thing as:

```
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "python"))
```

Reply:
That is indeed the old school way. pathlib is recommended now :-)

You could spell this with pathlib though:

```python
sys.path.insert(0, pathlib.Path(__file__).parent / "../python")
```

Personally, I do like something that is robust regardless of the exact path of the file itself. Another option is shelling out to git to find the root of the repository.

Reply:
"recommended" is a strong word when it comes to these parts of python.

Given all of the things that can break, I just swallow my objections and make it path dependent. When it is wrong, out is obvious and it didn't presume things it shouldn't.